perf-harness/src/test/java/io/opentelemetry/perf/OtlpPipelineStressTest.java

```diff
@@ -67,7 +67,7 @@
   public static GenericContainer<?> collectorContainer =
       new GenericContainer<>(
               DockerImageName.parse(
-                  "open-telemetry-docker-dev.bintray.io/java-test-containers:otel-collector-dev"))
+                  "ghcr.io/open-telemetry/java-test-containers:otel-collector-dev"))
```

Comment:
This change seems unrelated to the PR purpose. What is the reason for using a different registry?

Reply:
For a long time ever since starting to use bintray for agent smoke tests, we found the push reliability to be close to 0. GHCR has yet to fail :)

I had already updated our mirror workflow

https://github.com/open-telemetry/opentelemetry-java/pull/2161

But forgot to update the tests after that run once.

It's indeed not strongly related to the purpose but touches the same lines mostly, please let me be a bit lazy with it still being a small PR :D